app/mailers/user_emailer.rb

```diff
@@ -0,0 +1,11 @@
+class UserEmailer < ApplicationMailer
+  default from: 'laurawhalin@gmail.com'
```

Comment:
I think this may be the same as Chris' question above but since this inherits from ApplicationMailer does this need to have a "default from" in both?